src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -72,6 +74,52 @@ public static MeterProviderBuilder SetResourceBuilder(this MeterProviderBuilder
             return meterProviderBuilder;
         }
 
+        /// <summary>
+        /// Add view configuration.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <param name="meterName">Meter name.</param>
+        /// <param name="meterVersion">Meter version.</param>
+        /// <param name="instrumentName">Instrument name.</param>
+        /// <param name="instrumentKind">Instrument kind.</param>
+        /// <param name="aggregator">Aggregator to use.</param>
+        /// <param name="aggregatorParam">Aggregator parameter (Optional).</param>
+        /// <param name="viewName">View name (Optional).</param>
+        /// <param name="viewDescription">View description (Optional).</param>
+        /// <param name="attributeKeys">Attribute keys to include (Optional).</param>
+        /// <param name="extraDimensions">Attribute keys from baggage/context to include (Optional).</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProviderBuilder AddView(
+            this MeterProviderBuilder meterProviderBuilder,
+            string meterName = null,
+            string meterVersion = null,
+            string instrumentName = null,
+            string instrumentKind = null,
+            Aggregator aggregator = Aggregator.SUMMARY,
+            object aggregatorParam = null,
```

Comment:
Is it still the plan to leave the functionality for configuring aggregation out of the public API for now?

Reply:
at least for initial few releases

Reply:
I though we are just limiting the aggregator support to just one (i.e. Sum aggregator)?